common/README.md

```diff
@@ -15,3 +15,14 @@ pipenv install -e ./../common
 
 ### Run Tests
 `pipenv run tests` will run all tests.
+
+### Coverage
+`pipenv run unittests-cov` will run all unit tests with coverage enabled. \
+`pipenv run coverage-report-xml` will generate an xml file which can then be submitted for coverage analysis. \
+
+### Analysis
+To use sonarqube analysis you will need to have installed `sonar-scanner`. \
+Ensure sonar-scanner is on your path, and configured for the sonarqube host with appropriate token. \
```

Comment:
How do you do this configuration? If there's a webpage that explains this, then can you link to that?